app.R

```diff
@@ -0,0 +1,1882 @@
+                p("Diagnostics for stationarity include looking for constant mean
+               (or, trend) and variance over time"),
+               tags$ul(
+                 tags$li("Constant mean is associated with data that does not have any sort of vertical (typically linear) trend over time."),
+                 tags$li("Seasonality could also be apparent in the mean structure. Recall that seasonal ARIMA cannot explain a seasonal trend, only seasonal correlations (ARIMA models work to explain correlation structure of a time series AFTER the mean and variance are constant)."),
+                 tags$li("Constant variance is associated with data whose vertical spread (in the valleys and peaks) is constant over the duration of the time series.")
+               ), 
+               
+               h3("Autocorrelation Functions of Stationary Time Series:"),
+               p("We typically trust the dashed lines in the autocorrelation function
+               (ACF) plots to be the significance cut-off bounds for any lag's
+               correlation"),
+               p("In a model with non-zero autoregressive (AR) and moving average
+               (MA) parts, there is no logical interpretation for both ACFS cutting
+               off, thus,"),
+               tags$ul(
+                 tags$li("For AR(p) models, the ACF will tail off and the PACF will
+                      cut off after lag p."),
+                 tags$li("For MA(q) models, the ACF will cut off after lag q, and
+                      the PACF will tail off."),
+                 tags$li("For ARMA(p, q) models, both the ACF and the PACF will both
+                      tail off.")
+               ),
+               p("The ARMA subsets plot is not the best tool for determining ARMA(p,q)
+               orders, and thus will only be used as a tie breaker or guide after
+               the ACF and PACF plots have been thoroughly inspected."),
+               br(), 
+               h3("Model Diagnostics:"),
+               p("The ARIMA model aims to forecast future values of a stationary time
+              series by estimating a mathematical function to explain the underlying
+              correlation structure. For this reason, the ACF and PACF of the residuals
+              of the ARIMA model that has been fitted should not contain any significant
+              remaing correlation."),
+              p("Though forecasting is the purpose for fitting an ARIMA model, looking at the forecast itself (against future values that have been reserved) isnt the best way to assess the goodness of the model's fit, this is why we look at the AIC and the ACF plots of the residuals of the model."),
+              
+              br(),
+              br(), 
+              
+              div(style = "text-align: center",
+                  bsButton(
+                    inputId = "go1", 
+                    label = "GO!", 
+                    icon("wpexplorer"), 
```

Comment:
All of the go buttons except the first one should have the icon changed to the bolt.

Reply:
I did this because I remember one of the professors saying to make the symbol to the page the button goes to. So, I am not quite sure which is correct.